SYCL/BFloat16/bfloat16_example.cpp

```diff
@@ -14,19 +14,14 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend "-device gen9" %s -o %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
-
```

Comment:
Why are we removing the AOT GPU tests?  Is it because the CI system might not have a gen9 GPU?  If that is the case, can we somehow conditionally run the test only if the CI system is gen9?

Reply:
There isn't a way to check/select a particular Intel GPU in the CI infrastructure. The scenarios in the test are covered by the clang/Driver lit tests which check that the proper libraries are used. I could add a check in the test to check GPU type, but narrowing the scope of runnable tests isn't going to necessarily increase coverage, if the expected behavior has been checked in other ways.

Reply:
Correction. There does appear to be a way to select a specific Intel GPU. Let me try using that feature.